src/server/test/web/obviusConfig.js

```diff
@@ -0,0 +1,78 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/* This file tests the API for retrieving meters, by artificially
+ * inserting meters prior to executing the test code. */
+
+// The test line below does not work if you cannot properly connect
+// to the database.
+// let { chai, mocha, expect, app, testDB, testUser } = require('../common');
+mocha = require('mocha');
+chai = require('chai');
+const ini = require('ini');
+const fs = require('fs');
+const Meter = require('../../models/Meter');
+const ConfigFile = require('../../models/obvius/Configfile');
+
+const { log } = require('../../log');
+
```

Comment:
I'm a little unclear about several aspects of the tests in this file. They seem to test functions but then output to the logging system. First, I don't know if doing that is what is desired and is not a good way to test results. Second, I'm not clear what trigger causes the test to fail other than an error. Finally, there are some commented out lines that should either be removed or integrated into the code.